plugins/meson.build

```diff
@@ -12,6 +12,8 @@ plugin_deps = [
   protobufc,
 ]
 
+plugin_inputs = fwupdplugin_struct_targets
```

Comment:
got a better name than `inputs`? maybe we rename `fwupdplugin_struct_targets` to `fwupdplugin_rs` and just use that directly?

Reply:
I'm totally good with doing that.

Reply:
Done